client/Components/molocules/navigation/Navigation.jsx

```diff
@@ -70,25 +81,27 @@ export class Navigation extends Component {
 
                                         <ul className="rd-navbar-nav">
                                             <li className="rd-nav-item active"><Link to="/"><a className="rd-nav-link">Home</a></Link></li>
-                                            <li className="rd-nav-item active"><Link to="/lectures"><a className="rd-nav-link">Lectures</a></Link></li>
+                                            {/* <li className="rd-nav-item active"><Link to="/lectures"><a className="rd-nav-link">Lectures</a></Link></li> */}
```

Comment:
if not needed, please remove commented code

Reply:
it may be possible that we may use this code in future that i commented it

Reply:
ok, I see, no proble, just leave it there then